internal/cmd/oohelperd/http.go

```diff
@@ -102,13 +106,56 @@ func httpDo(ctx context.Context, config *httpConfig) {
 	reader := &io.LimitedReader{R: resp.Body, N: config.MaxAcceptableBody}
 	data, err := netxlite.ReadAllContext(ctx, reader)
 	ol.Stop(err)
+
+	h3Endpoint := ""
+	if !config.h3 {
+		h3Endpoint = getHTTP3Altsvc(resp)
+	}
+
 	config.Out <- ctrlHTTPResponse{
-		BodyLength: int64(len(data)),
-		Failure:    httpMapFailure(err),
-		StatusCode: int64(resp.StatusCode),
-		Headers:    headers,
-		Title:      measurexlite.WebGetTitle(string(data)),
+		BodyLength:   int64(len(data)),
+		DiscoveredH3: h3Endpoint,
```

Comment:
I think `DiscoveredH3Endpoint` has to be public, since it is defined in `internal/model` but used in `internal/oohelperd`, no?

Reply:
> I think `DiscoveredH3Endpoint` has to be public, since it is defined in `internal/model` but used in `internal/oohelperd`, no?

Indeed!

What do you think of renaming it?

Reply:
I think renaming makes sense and I already changed it to `DiscoveredH3Endpoint` :)

Reply:
Thanks!